pkg/zappr/client.go

```diff
@@ -144,6 +144,10 @@ func (c *clientImpl) doRequest(req *http.Request) (int, *zapprErrorResponse, err
 	zapprErrorResponse := &zapprErrorResponse{}
 	resp, err := c.slingClient.Do(req, nil, zapprErrorResponse)
 
+	if resp == nil {
+		return http.StatusInternalServerError, nil, errwrap.Wrap(ErrZapprServerError, err)
```

Comment:
you assume that if you don't get any response it always internal server error code, maybe it should be a different code for those situations?

Reply:
I just picked that as an "i don't know the status" option, which one do you suggest ?